### ISSUE TITLE
[rule] Bug in tag-close

### ISSUE THREAD

**Issue by acalvo** [linthtml/linthtml#68]
This code is reporting a "Tag is not closed" error.

```html
<span>&#8599;</span>
```
